UKHomeOffice/cop-private-workflow-tasklist
# COP-1164 Add dashboard banner

**AC**

- Add top banner section with purple background : #8F23B3 is HO Purple
- Make a 2/3 + 1/3 split
- Update H1 to Welcome to the Central Operations Platform
- Keep the two shift buttons
- Add divider lines to panels on mobile

**To Test**

Based on https://cop.prototype.cop.homeoffice.gov.uk/blue/sprint-34/dashboard-v0-3?nginxId=6c85fcf0-068e-402c-c026-3712c62e615c

- log into COP
- ensure you are on /dashboard (Home)
> the page should look similar to the prototype above with the following exceptions
-- the background is purple
-- the width is 1400px
-- there is no search functionality
-- the black panel contains only the users name & shift buttons

- resize browser
> the page should be responsive and collapse to a single column on mobile widths

- edit your shift
> the green confirmation of shift started should appear above the purple banner

Note: There is a bug that exists in Production that has not been fixed as part of this PR (COP-1473 in Jira)
